coding/python_style_guide.rst

```diff
@@ -619,21 +619,23 @@ Within a module, follow the order:
 
 .. _style-guide-py-super:
 
-``super`` SHOULD NOT be used unless the author really understands the implications (e.g. in a well-understood multiple inheritance hierarchy).
-----------------------------------------------------------------------------------------------------------------------------------------------
+``super`` MAY be used with care to call inherited methods.
+----------------------------------------------------------
 
-Python provides :py:func:`super` so that each parent class' method is only called once.
+Python provides ``super()`` so that each base class’s method is only called once.
```

Comment:
Does ``:py:func:`super` `` produce better output, semantically even if not visually?

Reply:
Nice! That actually links it. Thanks for that one.